src/daft-local-execution/Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [dependencies]
+arrow2 = {workspace = true}
```

Comment:
what do we need from the arrow2 crate? if it's the BitMap It may make sense to `re-export` it via `daft-core`